OSDOCS-5049 (Releasae note): Updated terminology for "whitelist" and "blacklist" annotations (#86277)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

[id="ocp-release-notes-networking_{context}"]
=== Networking

[id="ocp-release-notes-networking-whitelist-blacklist-annotation-updated-allowlist-denylist_{context}"]
=== Updated terminology for "whitelist" and "blacklist" annotations

> The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been updated to `ip_allowlist` and `ip_denylist`, respectively. Currently, {product-name} still supports the `ip_whitelist` and `ip_blacklist` annotations. However, these annotations will be removed in a future release.

Reviewer: ```suggestion
The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been updated to `ip_allowlist` and `ip_denylist`, respectively. Currently, {product-name} still supports the `ip_whitelist` and `ip_blacklist` annotations. However, these annotations are planned for removal in a future release.
```

Reviewer: I've always gotten the feedback to say "planned" instead of "will be" when we promise things in certain releases. I certainly hope these annotations will be removed, but just thought I'd comment my feedback anyways.

Reviewer: https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Author: That's awesome. Thanks for letting me know.
